<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Read_Codes</class>
 <widget class="QWidget" name="Read_Codes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Считывание штрих-кодов</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Введенный код:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtCode"/>
    </item>
    <item>
     <widget class="QLabel" name="lblKorob">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="lstInside"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGenerateAgr">
        <property name="text">
         <string>Сформировать файл агрегации</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
